replication/logic/replication_logic.go

```diff
@@ -453,10 +466,11 @@ func (fs *Filesystem) doPlanning(ctx context.Context) ([]*Step, error) {
 			})
 		}
 	} else { // resumeToken == nil
+		var initialSend = false
```

Comment:
```suggestion
		var initialSend bool
```

All Go types defaults to their zero-value for the type. false for bool, 0 for numbers (int, float, etc..), empty string for strings, nil for pointers etc...